.github/workflows/upload-release-image.yml.sample

```diff
@@ -0,0 +1,67 @@
+# This workflow dispatch is to be used on a downstream ansible-slurm-appliance repository. The workflow takes two inputs:
+# image name, and s3 bucket name. The image is searched for in the environment set openstack, and if not found there, 
+# downloads it from the ARCUS S3 bucket specified. The workflow then uploads the image to the openstack.
+#
+# To use this workflow in a downstream ansible-slurm-appliance repository simply copy it into .github/workflows
+# and give it an appropriate name, e.g.
+# cp .github/workflows/upload-s3-image.yml.sample .github/workflows/upload-s3-image.yml
+#
+# In order for the workflow to access the openstack, credentials in the form of a clouds.yaml file must be provided by a
+# secret. This secret should have the name OS_CLOUD_YAML to be found by the workflow.
+# Details on the contents of the clouds.yaml file can be found at https://docs.openstack.org/keystone/latest/user/application_credentials.html
+
+name: Upload release images to client sites from s3
+on:
+  workflow_dispatch:
+    inputs:
+      image_name:
+        type: string
+        description: Image name from https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_3a06571936a0424bb40bc5c672c4ccb1/{BUCKET_NAME}/
```

Comment:
It might be worth checking if we can use Markdown [syntax](https://www.markdownguide.org/basic-syntax/#links) for links here e.g. `Image name from [release bucket](https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_3a06571936a0424bb40bc5c672c4ccb1/openhpc-images/) or [pre-release bucket](https://object.arcus.openstack.hpc.cam.ac.uk/swift/v1/AUTH_3a06571936a0424bb40bc5c672c4ccb1/openhpc-images-prerelease/).` I'm not sure if GitHub UI supports that so maybe just give it a go.

Reply:
I don't think GitHub UI supports rich formatting in workflow_dispatch inputs.